_posts/en/2019-06-14-exec-briefing.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Bitcoin Optech 2019 Executive Briefing
+permalink: /en/2019-exec-briefing/
+name: 2019-06-14-exec-briefing
+type: posts
+layout: post
+lang: en
+
+excerpt: >
+  Videos and slides from the Bitcoin Optech Executive Briefing
+  on May 15th, 2019.
+
+---
+<style>
+/* For video embeds */
+.container {
+    position: relative;
+    width: 100%;
+    height: 0;
+    padding-bottom: 56.25%;
+}
+.video {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+}
+</style>
+
+{% include references.md %}
+
+On May 15th, 2019, Bitcoin Optech hosted an Executive Briefing session at the
+Chaincode Labs office in New York.  The seminars were targeted at executives and
+founders of Bitcoin companies, and covered what we think are the most important
+technical topics that executives needed to be aware of in 2019:
+
+- [__A Return to Fees__](#a-return-to-fees) - why fees spike in Bitcoin, why
+  high fees may return, and how businesses can thrive in a high-fee
+  environment.
+- [__Operating on Lightning__](#operating-on-lightning) - mainnet Lightning
+  Network capacity has exploded in the last 12 months. A survey of how
+  Lightning is being used now and discussion of opportunities for Bitcoin
+  businesses adopting Lightning.
+- [__The Next Softfork__](#the-next-softfork) - features of upcoming Bitcoin
+  softfork proposals: Schnorr signatures, Taproot and SIGHASH. What those
+  changes could mean for businesses.
+
+## A return to fees
+
+The first presentation was given by Bitcoin Optech contributor Mike
+Schmidt, and focused on transaction fees and ways to mitigate costs
+and user confusion.
+
+<div class="container">
+<iframe src="//www.youtube.com/embed/ihUQ4C42KUk"
+frameborder="0" allowfullscreen class="video"></iframe>
+</div>
+
+[Download slides][fees slides]
```

Comment:
I think this is a bit hard to see.  It might be a bit more visible if centered:

```
[Download slides][fee slides]
{:.center}
```

And you might want to throw a `<br>` after the iframe or tweak the div's CSS to include some bottom margin so that there's some separation between the video and the link text.

Reply:
Done